app/user/User.py

```diff
@@ -27,7 +26,7 @@ def __init__(
         self.hasUsedReferralCode = hasUsedReferralCode
 
     @staticmethod
-    def from_dict(email: str, dict):
+    def from_dict(email: str, dict: dict):
```

Comment:
nit: Missing return type. Also plz use types from `typing`.
`from typing import Dict, Any`
```suggestion
    def from_dict(email: str, dict: Dict[str, Any]) -> "User":
```
There's a couple other instances with this too.